src/helper/args/mod.rs

```diff
@@ -19,69 +19,67 @@ const DEBUG_EMOTICON: &str = "(o_O)";
 /// Separator for output.
 const OUTPUT_SEPARATOR: &str = "---";
 
-/// Checks if the given arguments exist.
-fn check_args<'a, ArgsIter: Iterator<Item = &'a str>, Output: Write>(
+/// Checks if the given argument exist.
+fn check_arg<Output: Write>(
     cmd: &str,
-    args: ArgsIter,
+    arg: &str,
```

Comment:
What was the design choice behind removing the iterator and using a str here?

Reply:
'cause I moved the iteration loop to the `get_args_help` so I can have more control over when to stop and when to continue